.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -25,14 +25,21 @@ body:
         - Queues
         - R2
         - Workers for Platforms
-        - Workers/Other
-        - Wrangler
+        - Constellation
+        - Wrangler core
+        - Other
```

Comment:
Could we add a `Workers Runtime` option here? (They'll probably all be shifted immediately over to ``workerd`, but we get a bunch of tickets related to that, and it would help with triage)

Reply:
👍